Error while building with GO111MODULE=on

`ambiguous import:` error while building/installing with `GO111MODULE=on`:

```
../../go/pkg/mod/github.com/edgexfoundry/go-mod-core-contracts@v0.1.3/models/event.go:19:2: unknown import path "github.com/ugorji/go/codec": ambiguous import: found github.com/ugorji/go/codec in multiple modules:
	github.com/ugorji/go v1.1.4 (/Users/acourouble/go/pkg/mod/github.com/ugorji/go@v1.1.4/codec)
	github.com/ugorji/go/codec v0.0.0-20181204163529-d75b2dcb6bc8 (/Users/acourouble/go/pkg/mod/github.com/ugorji/go/codec@v0.0.0-20181204163529-d75b2dcb6bc8)
```
It seems like edgexfoundry/go-mod-core-contracts is dependent on `github.com/ugorji/go v1.1.4`.

And cobra is dependent on `github.com/ugorji/go/codec v0.0.0-20181204163529-d75b2dcb6bc8`.

This error breaks building with `GO111MODULE=on`. The current workaround is to build and install with `GO111MODULE=off`.

--- a/README.md
+++ b/README.md
@@ -15,7 +15,7 @@ $ cd ~/go/src/github.com/edgexfoundry/
 $ git clone https://github.com/edgexfoundry-holding/edgex-cli
 ```
 
-*A current bug prevents usage with go mod. The workaround is to download in go path and to build/install with GO111MODULE=off. This bug is described in [issue #26](https://github.com/edgexfoundry-holding/edgex-cli/issues/26). make install and make build work.*
+*A bug described in [issue #26](https://github.com/edgexfoundry-holding/edgex-cli/issues/26) is solved through the use of the replace directive. This fix, however, can be erased when using go tidy and some other go commands. make install and make build work.*
 
 * Install the CLI:
 

--- a/go.mod
+++ b/go.mod
@@ -4,3 +4,5 @@ require (
 	github.com/edgexfoundry/go-mod-core-contracts v0.1.3
 	github.com/spf13/cobra v0.0.5
 )
+
+replace github.com/ugorji/go v1.1.4 => github.com/ugorji/go/codec v0.0.0-20190204201341-e444a5086c43

